### ISSUE TITLE
Should download button in docs open down when there's space?

### ISSUE THREAD

**Issue by jorgeorpinel** [iterative/dvc.org#1162]
I get that it opens up when the nav is really long like in the cmd ref, but when you first open docs there's usually space, so it could open down.

![image](https://user-images.githubusercontent.com/1477535/79538909-dc4a1800-804a-11ea-82b2-59204a21ccd6.png)

**Comment by jorgeorpinel** [comment]
Ping @iterative/websites WDYT? Probably a simple change: I just modified this in the browser to force it under:

![image](https://user-images.githubusercontent.com/1477535/158291090-d0f0b4b9-8333-4864-bbb5-7409f31ca270.png)

![image](https://user-images.githubusercontent.com/1477535/158291147-76837a8f-a399-4809-ba6f-56df4f2a031f.png)

**Comment by julieg18** [comment]
> Just needs the logic to check whether there's space in the window.

Yes, this task is doable with a little JS! Not sure if its `p2` though :)

**Comment by jorgeorpinel** [comment]
Feel free to prioritize within your team as appropriate. Thanks
